pjlib/include/pj/config.h

```diff
@@ -933,6 +916,20 @@
 #   define PJ_HAS_STRICMP_ALNUM	    0
 #endif
 
+/*
+ * Warn about obsolete macros.
+ *
+ * PJ_ENABLE_EXTRA_CHECK has been deprecated in 2.13.
+ */
+#if defined(PJ_ENABLE_EXTRA_CHECK)
```

Comment:
If user explicitly enable it, I think we don't need to give any warning to minimise any inconvenience (user may need to remove the setting and rebuild the entire library). The behavior doesn't change anyway.
So we should only give warning if PJ_ENABLE_EXTRA_CHECK==0.